cmd/oras/internal/option/common.go

```diff
@@ -28,23 +28,22 @@ const NoTTYFlag = "no-tty"
 
 // Common option struct.
 type Common struct {
-	Debug   bool
-	Verbose bool
-	TTY     *os.File
 	*output.Printer
```

Comment:
Can you make it `Printer *output.Printer`?

Reply:
That would touch too many places though, as there are many `opts.Println` in the code base.